### ISSUE TITLE
how to set mgroup span across columns when output to rtf?

### ISSUE THREAD

**Issue by fredericky123** [benjann/estout#50]
```
eststo clear
sysuse auto,clear
eststo:reg rep78 weight length
eststo:reg rep78 weight 
eststo:reg rep78  length
eststo:reg rep78 weight length
eststo:reg rep78 weight
eststo:reg rep78  length
esttab, mgroups("Female evaluators" "Male evaluators" "All evaluators", pattern(1 0 1 0 1 0) span)
```

<img width="882" alt="image" src="https://user-images.githubusercontent.com/13982749/203058607-51058659-760d-42fd-87dd-1d44611f54e0.png">
How to set the label groups of models across columns?
Thanks!

**Comment by benjann** [comment]
I am afraid there is no automatic way to do this (at least not for screen display; in LaTeX this is easier, see [http://repec.sowi.unibe.ch/stata/estout/latex.html](http://repec.sowi.unibe.ch/stata/estout/latex.html)).

Here are a few suggestion for how to center the titles manually:

```Stata
sysuse auto
regress price weight
est sto ols

esttab ols ols ols ols, ///
    mgroups("          Female evaluators" ///
            "          Male evaluators", pattern(1 0 1 0) span)

esttab ols ols ols ols, ///
    mgroups("Female evaluators" ///
            "Male evaluators", prefix("          ") pattern(1 0 1 0) span)
            
esttab ols ols ols ols, ///
    mgroups("{space 10}Female evaluators" ///
            "{space 10}Male evaluators", pattern(1 0 1 0) span)

esttab ols ols ols ols, ///
    mgroups("Female evaluators" ///
            "Male evaluators", prefix("{space 10}") pattern(1 0 1 0) span)

esttab ols ols ols ols, ///
    mgroups("Female evaluators" ///
            "Male evaluators", ///
            prefix("{space 3}{center 28:") suffix("}") pattern(1 0 1 0) span) asis
```

How do I get to `{space 3}{center 28}`? Each column has a width of 12 by default, the stars consume 3, and there is a blank between columns of 1. I.e. 28 = 2*12+3+1
ben

**Comment by fredericky123** [comment]
Thanks so much, Ben!
This can be an alternative!

**Comment by NilsEnevoldsen** [comment]
Beat me to it. I'll just add that it's easier to see where the groups begin and end if you add a visible character to the beginning and the end, and shorten the corresponding centering width by two characters. I do this while mocking up the table, and then remove them when I get it right.

```
esttab, mgroups("Female evaluators" "Male evaluators" "All evaluators", pattern(1 0 1 0 1 0) span prefix(|{space 3}{center 26:) suffix(}|)
```

**Comment by fredericky123** [comment]
Thanks a ton for this tips!

**Comment by fredericky123** [comment]
However, I found this useful for screen display; not useful for rtf file. Is there a solution for rtf file?

**Comment by benjann** [comment]
Not possibly in rtf, I believe. You have to leave away the `span` suboption and then merge the cells manually in Word.

**Comment by fredericky123** [comment]
Alright, thanks!

**Comment by flxflks** [comment]
In this simple case, I have been able to use pandoc to span the column headers:
```
* Export the document to Latex first
esttab using example.tex, replace mgroups("Female evaluators" "Male evaluators" "All evaluators", pattern(1 0 1 0 1 0) span prefix(\multicolumn{@span}{c}{) suffix(}))

* Use pandoc to convert the document into docx
shell pandoc example.tex -o example.docx
```

EDIT: I found the error in my code—Pandoc seems to work fine, with one important caveat: I haven't yet figured out how to get \hline to render in the output and I do not think it is possible. Also, when using `&` in table content, it needs to be escaped with a backslash. For example, use `indicate("Country \& FE = *.year")` rather than `indicate("Country & FE = *.year")`.

I do not know what is faster: Manully adding the horizontal and vertical lines OR manually merging the cells. 
My hunch is that building more complex tables is probably easier in using pandoc, but I'd be curious to know what other people think.
